conftest.py

```diff
@@ -0,0 +1,31 @@
+from django.conf import settings
+
+
+def pytest_configure():
+    settings.configure(
+        DEBUG=True,
+        USE_TZ=True,
+        DATABASES={
+            "default": {
+                "ENGINE": "django.db.backends.sqlite3",
+            }
+        },
+        ROOT_URLCONF="test_urls",
+        INSTALLED_APPS=[
+            "django.contrib.auth",
+            "django.contrib.contenttypes",
+            "django.contrib.sites",
+            "django.contrib.sessions",
+            "djpyfs",
+        ],
+        MIDDLEWARE_CLASSES=[
+            'django.contrib.sessions.middleware.SessionMiddleware',
+            'restrictedsessions.middleware.RestrictedSessionsMiddleware',
+            'django.middleware.common.CommonMiddleware',
+            'django.middleware.csrf.CsrfViewMiddleware',
+            'django.contrib.auth.middleware.AuthenticationMiddleware',
+            'django.contrib.messages.middleware.MessageMiddleware',
+            'django.middleware.clickjacking.XFrameOptionsMiddleware',
+        ],
+        SITE_ID=1,
+    )
```

Comment:
Interesting way of doing the test settings.  I usually use a separate test_settings.py module to stick closer to the regular settings syntax, but this has the advantage of not requiring explicit specification of the settings module in tox.ini and elsewhere.  I suppose setting an environment variable here to specify the separate settings module would have the best features of both approaches, but there are few enough settings here there's no real need to change it again.